DEVELOPMENT.md

```diff
@@ -1,47 +1,44 @@
 # Developing the Tekton website
 
 - [Developing the Tekton website](#developing-the-tekton-website)
-  - [Running Locally](#running-locally)
-  - [tekton.dev](#tektondev)
+- [Running Locally](#running-locally)
+- [tekton.dev](#tektondev)
 
-## Running Locally
-
-
-The [tekton.dev](#tekton.dev) is deployed on Netlify. Netlify will invoke
-[the Makefile](.Makefile) to build the website.
-
-This is configured in:
-
-* [netlify.toml](netlify.toml)
-* [runtime.txt](runtime.txt) (the python runtime)
-* [requirements.txt](requirements.txt) (the python requirements)
-
-To run it locally you will need to install:
+## Dependencies
 
-* [Hugo](https://gohugo.io/) [version 0.53](netlify.toml)
-* `npm` to install Hugo's requirements:
-  * `npm install` (in directory with [package.json](package.json))
+* `python3` [installation link](https://www.python.org/downloads/) 
+* `hugo (EXTENDED VERSION)` [installation link](https://github.com/gohugoio/hugo/releases)
+* `pip` [installation link](https://pip.pypa.io/en/stable/installing/)
+* `npm v6.14.5` [installation link](https://nodejs.org/en/)
+* `node v14.3.0` [installation link](https://nodejs.org/en/)
+* `netlify cli` [installation link](https://cli.netlify.com/getting-started)
+* `netlify account` [sign up](https://app.netlify.com/)
 
-Then you can run [hugo commands](https://gohugo.io/getting-started/usage/) such as:
+## Running Locally
 
+Step 1
 ```bash
-hugo server
+git clone https://github.com/tektoncd/website && cd website
 ```
-
-To run the [Makefile](Makefile) you will need more dependencies and flags:
-
-* `python` aliased to python3.7 - [sync.py](sync/sync.py) is running `python` but assumes this is python3 (configured in [runtime.txt](runtime.txt))
-* `pip`, to install python requirements:
-  * Install with `pip install -r requirements.txt`
-
-To run the makefile locally, you'll have to set the env var:
-
-* [`DEPLOY_PRIME_URL`](https://docs.netlify.com/configure-builds/environment-variables/) to the path to your website files
-
-For example:
-
+Step 2
+```bash
+node i
+```
+Step 3
+```bash
+pip install -r requirements.txt
+```
+Step 4
+```bash
+alias python=python3
+```
+Step 5
+```bash
+python sync/sync.PY
```

Comment:
Instead of doing the alias, can we just recommend using pip3/python3 instead?

Reply:
I can 100% change it. I would also have to change the make file as well.

Reply:
Also, I thought that the alias was also really weird. I didnt want to break it cause I do not know what the CI/CD pipeline looks like for syncing. There could be a case where python3 doesnt exist and python does. For example
``` bash
alias python="/random/directory/my_python/python3.6
```
Changing it to python3 in make would be breaking.
I am going to change the make file so it is:

``` bash
python sync/sync.py || python3 sync/sync.py
```
So that way this change is not breaking.